torch/cuda/_utils.py

```diff
@@ -159,7 +208,7 @@ def check_nvrtc(result: int) -> None:
     check_nvrtc(libnvrtc.nvrtcGetPTX(prog, ptx))
     libnvrtc.nvrtcDestroyProgram(ctypes.byref(prog))
 
-    return ptx.value
+    return ptx.value if not torch.version.hip else ptx.raw[:ptx_size.value]
```

Comment:
nit: this is a bit hard to read, prefer not sure if its a pytorch convention to keep it in cur form?
```python
return ptx.raw if hip else ptx.value
```

Reply:
Changed it to `return ptx.raw if torch.version.hip else ptx.value.` Would this be okay?